README.md

```diff
@@ -2,65 +2,82 @@
 
 [![CI](https://github.com/NYULibraries/spatial_data_repository/actions/workflows/ci.yml/badge.svg)](https://github.com/NYULibraries/spatial_data_repository/actions/workflows/ci.yml)
 
-This repository tracks the implementation of GeoBlacklight used to power the front-end of NYU Libraries [Spatial Data Repository](https://geo.nyu.edu)
+This repository tracks the implementation of GeoBlacklight used to power the front-end of NYU Libraries' [Spatial Data Repository](https://geo.nyu.edu)
 
-## Links
```

Comment:
Moved Links & Archive into a Resources section below. Let's get folks up and running with the app, then possibly send them off to other documentation.